css/_sidenavs.scss

```diff
@@ -110,3 +110,14 @@
     margin-right: 10px!important;
     margin-left: 5px!important;
 }
+
+md-sidenav {
```

Comment:
Yes. Actually I started off using a class called `command-panel` or something. But later I changed because and we don't need to change templates if we use tags instead. I totally agree with that point in the MDN guide.